src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor(), new Random().nextInt(100));
```

Comment:
same, lets make Random and ColorSupplier as class level variables, not to create new instances every time we call getRandomBall() method. Also lets make number 100 as constant field https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names